src/lib.rs

```diff
@@ -872,7 +786,14 @@ impl CommonMarkViewerInternal {
                     text: String::new(),
                 });
             }
-            pulldown_cmark::Tag::Image(_, url, _) => self.start_image(url.to_string(), ui, cache),
+            pulldown_cmark::Tag::Image(_, url, _) => {
+                let url = if url.starts_with("http://") || url.starts_with("https://") {
+                    url.to_string()
+                } else {
+                    format!("file://{url}")
```

Comment:
I think this was a mistake. I think we should pass the uri unchanged to egui.

For instance, the user might specify their uri using something like `bytes://` to indicate it should come from the `BytesLoader`. Or they specify it using `file://` and now `egui_commonmark` adds another `file://` prefix to it